tools/Makefile.sub

```diff
@@ -8,10 +8,15 @@ jlc_SOURCES = \
 jlc_LIBS += \
 	libtooling \
 	libllvm \
+	libmlir \
 	librvsdg \
 	libutil \
 
-jlc_EXTRA_LDFLAGS += $(shell $(LLVMCONFIG) --libs core irReader) $(shell $(LLVMCONFIG) --ldflags) $(shell $(LLVMCONFIG) --system-libs)
+jlc_EXTRA_LDFLAGS += \
+	       $(shell $(LLVMCONFIG) --libs core irReader) \
+	       $(shell $(LLVMCONFIG) --ldflags) \
+	       $(shell $(LLVMCONFIG) --system-libs) \
+	       ${MLIR_LDFLAGS} \
```

Comment:
Why does jlc all in a sudden have a dependency on libmlir (and because of that on the MLIR_LDFLAGS)?

Reply:
This has been discussed earlier and is item 3 in issue https://github.com/phate/jlm/issues/379